docs/dev.rst

```diff
@@ -5,37 +5,70 @@ Open edX development
 
 In addition to running Open edX in production, Tutor can be used for local development of Open edX. This means that it is possible to hack on Open edX without setting up a Virtual Machine. Essentially, this replaces the devstack provided by edX.
 
-The following commands assume you have previously launched a :ref:`local <local>` Open edX platform. If you have not done so already, you should run::
+.. warning::
 
-    tutor local quickstart
+  Do not run ``tutor dev ...`` commands on a production system. They can modify your configuration and database in ways that are nonsensical or insecure for production use.
 
-To run the platform in development mode, you **must** answer no ("n") to the question "Are you configuring a production platform".
 
-Note that the local.overhang.io `domain <https://dnschecker.org/#A/local.overhang.io>`__ and its `subdomains <https://dnschecker.org/#CNAME/studio.local.overhang.io>`__ all point to 127.0.0.1. This is just a domain name that was set up to conveniently access a locally running Open edX platform.
+First-time setup
+----------------
 
-Once the local platform has been configured, you should stop it so that it does not interfere with the development environment::
+Start by launching a development platform::
 
-    tutor local stop
+    tutor dev quickstart
 
-Finally, you should build the ``openedx-dev`` docker image::
+This will perform several tasks for you. It will:
 
-    tutor dev dc build lms
+* stop any existing locally-running Tutor containers,
 
-This ``openedx-dev`` development image differs from the ``openedx`` production image:
+* disable HTTPS,
 
-- The user that runs inside the container has the same UID as the user on the host, to avoid permission problems inside mounted volumes (and in particular in the edx-platform repository).
-- Additional python and system requirements are installed for convenient debugging: `ipython <https://ipython.org/>`__, `ipdb <https://pypi.org/project/ipdb/>`__, vim, telnet.
-- The edx-platform `development requirements <https://github.com/openedx/edx-platform/blob/open-release/maple.master/requirements/edx/development.in>`__ are installed.
+* set your ``LMS_HOST`` to `local.overhang.io`_ (a conveninece domain that simply `points at 127.0.0.1`_),
+
+* prompt for a platform details (with suitable defaults),
+
+* offer to import the demo course,
```

Comment:
I'd rather not do that -- and also not the superuser creation below. Provisioning should be done manually by the end user.

Reply:
OK 👍🏻